meson_options.txt

```diff
@@ -3,6 +3,11 @@ option('default-backend',
 	value: '',
 	description: 'Name of the backend library to load, instead of libWPEBackend-default.so'
 )
+option('enable-fs',
+	type: 'boolean',
+	value: true,
```

Comment:
Would want to be `false` before landing but putting as `true` to make sure the build is successful.